app/main.js

```diff
@@ -479,6 +469,11 @@ function saveDefaultsFor (array, next) {
 
 function getTrayMenu () {
   let trayMenu = []
+  let timeLeft = breakPlanner.scheduler.timeLeft
```

Comment:
can we moved this `minutes` creation to utils or somewhere? we could also test it out there with a simple test ;]